src/pages/periodType.tsx

```diff
@@ -35,7 +35,7 @@ const PeriodType = ({ error }: PeriodTypeInterface): ReactElement => {
                                         id="periodtype-geo-zone"
                                         name="periodType"
                                         type="radio"
-                                        value="geozone"
+                                        value="periodGeoZone"
```

Comment:
We'll need to update the api to handle these new values for the redirects as well